templates/rundeck-config.erb

```diff
@@ -32,8 +32,13 @@ grails.mail.default.from = "<%= @mail_config['defaults.from'] %>"
 <%- end -%>
 grails.serverURL = "<%= @grails_server_url %>"
 rundeck.clusterMode.enabled = "<%= @clustermode_enabled %>"
-<%- if @file_keystorage_dir %>
-rundeck.storage.provider."1".config.baseDir = "<%= @file_keystorage_dir %>"
-rundeck.storage.provider."1".path = "/"
+
+rundeck.projectsStorageType = "<%= @projects_storage_type %>"
```

Comment:
Don't you also need to add other project storage config here .e.g. `rundeck.config.storage.provider."1"` ?